.travis.yml

```diff
@@ -1,7 +1,29 @@
+sudo: required
+
 language: node_js
 node_js:
-  - "stable"
+- stable
+
+services:
+- docker
+
+before_install:
+- docker build -t kevinchu/first-blog-mern .
+- docker run -d -p 127.0.0.1:4000:3000 kevinchu/first-blog-mern
```

Comment:
What are you trying to achieve here by running the image?

Reply:
removed, I have no reason to run the image.